Source/SpatialGDK/Private/SpatialGameInstance.cpp

```diff
@@ -62,8 +62,11 @@ bool USpatialGameInstance::StartGameInstance_SpatialGDKClient(FString& Error)
 		GetEngine()->ShutdownWorldNetDriver(GetWorldContext()->World());
 	}
 
-	FURL BaseURL = WorldContext->LastURL;
-	FURL URL(&BaseURL, TEXT("127.0.0.1"), ETravelType::TRAVEL_Absolute);
```

Comment:
This line turned out the be the source of the issue. Creating a new URL in this way would cause the GameDefaultMap to be set as the PIE map for this URL. We don't actually require an explicit declaration of the hostname since it defaults to local.